openstates/va/__init__.py

```diff
@@ -1,185 +1,79 @@
-        '2015 Special Session I',
-        '2015 Session',
-        '2014 Special Session I',
-        '2014 Session',
-        '2013 Special Session I',
-        '2013 Session',
-        '2012 Special Session I',
-        '2012 Session',
-        '2011 Special Session I',
-        '2011 Session',
-        '2010 Session',
-        '2009 Session',
-        '2009 Special Session I',
-        '2008 Session',
-        '2008 Special Session I',
-        '2008 Special Session II',
-        '2007 Session',
-        '2006 Session',
-        '2006 Special Session I',
-        '2005 Session',
-        '2004 Session',
-        '2004 Special Session I',
-        '2004 Special Session II',
-        '2003 Session',
-        '2002 Session',
-        '2001 Session',
-        '2001 Special Session I',
-        '2000 Session',
-        '1999 Session',
-        '1998 Session',
-        '1998 Special Session I',
-        '1997 Session',
-        '1996 Session',
-        '1995 Session',
-        '1994 Session',
-        '1994 Special Session I',
-        '1994 Special Session II',
+from pupa.scrape import Jurisdiction, Organization
+from .people import VaPersonScraper
+from .bills import VaBillScraper
+from .common import url_xpath
+
+
+class Va(Jurisdiction):
+    division_id = "ocd-division/country:us/state:va"
+    classification = "government"
+    name = "Virginia"
+    url = "http://virginiageneralassembly.gov/"
+    scrapers = {
+        "people": VaPersonScraper,
+        "bills": VaBillScraper,
+    }
+
+    legislative_sessions = [
+        {'name': '2010 Session', 'identifier': '101', 'classification': 'primary'},
+        {'name': '2011 Session', 'identifier': '111', 'classification': 'primary'},
+        {'name': '2011 Special Session I', 'identifier': '112', 'classification': 'special'},
+        {'name': '2012 Session', 'identifier': '121', 'classification': 'primary'},
+        {'name': '2012 Special Session I', 'identifier': '122', 'classification': 'special'},
+        {'name': '2013 Session', 'identifier': '131', 'classification': 'primary'},
+        {'name': '2013 Special Session I', 'identifier': '132', 'classification': 'special'},
+        {'name': '2014 Session', 'identifier': '141', 'classification': 'primary'},
+        {'name': '2014 Special Session I', 'identifier': '142', 'classification': 'special'},
+        {'name': '2015 Session', 'identifier': '151', 'classification': 'primary'},
+        {'name': '2015 Special Session I', 'identifier': '152', 'classification': 'special'},
+        {'name': '2016 Session', 'identifier': '161', 'classification': 'primary'},
+        {'name': '2017 Session', 'identifier': '171', 'classification': 'primary'},
     ]
-}
 
+    ignored_scraped_sessions = [
+        '001', '011', '012', '021', '031', '041', '042', '043', '051', '061',
+        '062', '071', '081', '082', '083', '091', '092', '941', '942', '943',
+        '951', '961', '971', '981', '982', '991'
+    ]
+
+    def get_organizations(self):
```

Comment:
we'll have to make sure this gets updated, on the pupa-nc branch I put in a new generic get_organizations template that may be a useful starting point